### DISCUSSION TITLE
Unable to access Scypted via Reverseproxy since 0.41.0

### DISCUSSION THREAD

**Discussion by DanielWeeber** [koush/scrypted#991] [Q&A] [answered]
Hey,

I am running Scrypted on HomeAssistant. I connect via NGINX Proxy Manager to HomeAssistant.
Everything is working, Node-Red, any other Addon, except Scrypted. If I click on the button in the menu it shows an error, see screenshot.
It was working before the update to 0.41.0 just fine.

If I open HomeAssistant via IP/Port instead of the DomainName from nginx proxy manager everything works as expected.

<img width="1369" alt="Bildschirm­foto 2023-07-31 um 13 08 58" src="https://github.com/koush/scrypted/assets/2935130/19b5c498-54ea-4b5f-a652-7134aae7846e">

**Comment by koush** [comment]
may want to check port https 10443 directly. and also check the container logs.

**Comment by DanielWeeber** [comment]
10443 directly is working
nothing in the container logs when I try to open via HomeAssistant

Its also working when I connect to HA via IP:8123 - just not when connecting via reverseproxy in front of HA.

**Comment by koush** [comment, marked as answer]
Given that direct ip/port works for both HA and Scrypted, this is an nginx routing issue.

**Comment by DanielWeeber** [comment]
Was anything changed in v0.41.0 in relation to ingress?

**Comment by koush** [comment]
Nope.

**Comment by DanielWeeber** [comment]
Just tested with another browser - working. Just had to delete Cookies in Chrome, now its working again..
